### PR TITLE
Include additional fields to manifest

### PR DESCRIPTION
**Pull request by jorgee**
Include new fields in the manifest and update the documentation with the new fields

### CODE REVIEW

File: docs/config.md [hunk 1]
```diff
@@ -1237,15 +1237,27 @@ The following settings are available:
...
 : Project related publication DOI identifier.
 
 `manifest.homePage`
 : Project home page URL.
 
+`manifest.iconUrl`
+: Project related icon location (Relative path or URL).
+
+`manifest.license`
+: Workflow license.
```
Comment (bentsherman) [inline docs/config.md:1253]:
```suggestion
: Project license.
```

**Comment by jorgee** [comment]
> Looks neat. Thin @ewels was suggesting to user `icon` instead of `iconUrl` in a recent comment

Yes, changed.

**Comment by pditommaso** [comment]
Well done! 🎉

**Comment by ewels** [comment]
Ooh awesome! Thanks @jorgee!

Looks like author / maintainer fields are just strings here. Could we optionally have these as maps as well with additional subfields such as ORCiD, affiliatiob and email? See https://github.com/nextflow-io/nextflow/issues/4034#issuecomment-1766364814

'cc @pditommaso @bentsherman

**Comment by pditommaso** [comment]
Think we said it would be better to keep it flat

**Comment by bentsherman** [comment]
The BCO supports a bunch of contributor metadata as a list of maps, so you can do something like name, email, ORCID

https://wiki.biocomputeobject.org/Provenance-domain#Contributors_%E2%80%9Ccontributors%E2%80%9D

I think we could support this structure directly in the config. The `manifest.author` could be either a string for comma-separated list, or a list of maps for the complete structure. I think name, affiliation, email, and ORCID should be enough, but always easy to add more fields later.

See also: https://github.com/nextflow-io/nf-prov/issues/10

**Comment by bentsherman** [comment]
So the list of maps is mainly to provide a more complete BCO manifest

**Comment by jorgee** [comment]
In fact, there is no check in the author, so you can put whatever valid expression (String, list, list of maps). 
```
manifest {
     author = [ [ name: 'First Author', email: 'author1@mail.com'], [ name: 'Second Author', email: 'author@mail.com']]
}
```
The above configuration should work. In the `nextflow info`, it will print the string serialization of the list of maps. 
I think we just need to validate the map fields.

**Comment by ewels** [comment]
Ok so if it already works, shall we say that a short term solution is to just modify the documentation to say that it can be a string or a map? Sounds like that would be sufficient for now. We can standardise on nf-core and come back to this if needed.

**Comment by ewels** [comment]
👉🏻  https://github.com/nextflow-io/nextflow/pull/5316

**Pull request merged**